packages/rpc-core/test/e2e/state.test.ts

```diff
@@ -2,10 +2,20 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { Balance, Bytes, Metadata, Moment } from '@polkadot/types';
+import fs from 'fs';
+import path from 'path';
+
+import { Balance, Bytes, Hash, Metadata, Moment, StorageData, StorageKey } from '@polkadot/types';
+import { Abi } from '@polkadot/api-contract';
 import storage from '@polkadot/storage/static';
 import WsProvider from '@polkadot/rpc-provider/ws';
 
+import ApiPromise from '@polkadot/api/promise/Api';
+import { SubmittableResult } from '@polkadot/api';
+import { KeyringPair } from '@polkadot/keyring/types';
+import testingPairs from '@polkadot/keyring/testingPairs';
+import flipperAbi from '../../../api-contract/test/contracts/flipper.json';
+
```

Comment:
(nit) Remove this extra space, goes with above one (which should have a blank line preceding it).